website/pages/docs/secrets/kv/index.mdx

```diff
@@ -13,6 +13,8 @@ modes. It can be a generic Key-Value store that stores one value for a key.
 Versioning can be enabled and a configurable number of versions for each key
 will be stored.
 
+Note that `kv` secrets mounts are case-sensitive.
```

Comment:
To comply with the use of "note" in other places, could we use this syntax to call it out in a box? 
```suggestion
~> **NOTE:** 
```